scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -372,7 +372,10 @@ function verify_subm_routeagent_container() {
 }
 
 function verify_subm_broker_secrets() {
-  # Show all SubM secrets
+  # Wait for secrets to be created
+  while ! kubectl get secret -n $subm_broker_ns | grep -q submariner-; do sleep 2; done
```

Comment:
This can wait forever theoretically, can't it ?

Reply:
Yeah, a time limit would be nice.

Reply:
Implemented in the latest version. :+1: